### PR TITLE
add eda

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**
Idk what to add, if you have any suggestions, slide into my dms

### CODE REVIEW

File: 01-switzerland-more-involved.ipynb [hunk 1]
```diff
@@ -0,0 +1,284 @@
+{
```
Comment (florinbarbisch) [inline 01-switzerland-more-involved.ipynb]: <p>Kann ihc nix lese</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/9/discussion/#comment-1598719588'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

File: utils.py [hunk 2]
```diff
@@ -2,128 +2,102 @@
...
-        del relationships
-
-        # save the graph
-        nx.write_gexf(G, GRAPH_PATH)
-
-        return G
-
-    return nx.read_gexf(GRAPH_PATH)
+    if os.path.exists(GRAPH_PATH):
+        return nx.read_gexf(GRAPH_PATH)
```
Comment (florinbarbisch) [inline utils.py:19]: Clean code right there! (returning instantly so the code below doesn't get indented too much)

**Review by florinbarbisch — DISMISSED** [review]
Merge the beast

**Comment by florinbarbisch** [comment]
Well not exactly: resolve unreadable plot problem first

**Review by florinbarbisch — CHANGES_REQUESTED** [review]
resolve unreadable plot problem first

File: 01-switzerland-more-involved.ipynb (hunk 1 above)
Comment (gabrieltorresgamez) [reply 01-switzerland-more-involved.ipynb]: <p>this are remnants of tobis branch, which I merged. Should I delete this file?</p>

File: utils.py (hunk 2 above)
Comment (gabrieltorresgamez) [reply utils.py:19]: 🥰🥰

**Comment by gabrieltorresgamez** [comment]
solved in the most spectacular way imaginable

**Pull request merged**